azuki-core/src/test/kotlin/com/anaplan/engineering/azuki/core/system/SemanticVersionFilterTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.anaplan.engineering.azuki.core.system
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class SemanticVersionFilterTest {
+
+    @Test
+    fun noScenarioVersion() {
+        val semanticVersionFilter = SemanticVersionFilter("1.0.0")
+        assertEquals(true, semanticVersionFilter.canVerify(null))
+        assertEquals(true, semanticVersionFilter.canVerify(null))
```

Comment:
⛏️ Why assert the same thing twice?

Reply:
There were two args originally, auto-refactor can't catch